### PR TITLE
Fix/color shows only after refresh

### PR DESCRIPTION
**Pull request by y-golde**
switched the ordering of ```setRows``` and the addition of the new randomized color so that the color will be added to the rows before the rows are re-set.

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -207,7 +207,7 @@ const InvestigationTable: React.FC = (): JSX.Element => {
         switch (cellName) {
             case TableHeadersNames.color:
                 return (
-                    indexedRow.groupId ?
+                    indexedRow.canFetchGroup ?
```
Comment (eladyashar) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:210]: why did you changed this condition ?
what its trying to check ?

Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:210]: I've noticed in another locaiton in the code that ```canFetchGroup``` exists 
this is it's declatation
image
It's just a boolean representation of if groupId (which is a number) exists.
I think this is a more _typescript_ appropriate approach to this condition

Comment (eladyashar) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:210]: cool

Comment (y-golde) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:210]: OK , So turns that when the row is a fetched row (linked row from the table) the value of canFetchGroup is false - so i changed it to Boolean(indexedRow.groupId) in the next commit 😞

**Review by MayaSasson — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

**Pull request merged**